.github/workflows/project-page-pull-request.yml

```diff
@@ -23,7 +23,9 @@ jobs:
         GH_TOKEN: ${{ github.token }}
       run: |
         title=$(gh issue view ${{ github.event.issue.number }} --repo ${{ github.repository }} --json title --jq '.title')
+        url=$${{ github.event.issue.html_url }}
         echo "title [$title]"
+        echo "url [$url]"
```

Comment:
Consider moving the added lines at the end of the block and add missing update of `GITHUB_OUTPUT`[^1]:

```
url=$${{ github.event.issue.html_url }}
echo "url [$url]"
echo "title=$title" >> $GITHUB_OUTPUT
```

[^1]: This ensures that subsequent steps can reference this step output

Reply:
I also suggest to keep the parameter named `html_url` here. It is only later that this specific metadata is referenced in a discussion section.

Reply:
are you saying to change the name url -> html_url ?